### PR TITLE
feat(content): Return to Paradise mission (adapted from #5112)

### PR DESCRIPTION
**Pull request by bene-dictator**
**Content (Mission)**

## Summary
This PR adapts the first mission from #5112 about bringing a stranded Paradise Worlds family back home, which was not related to the rest of the journalism missions but was part of that PR anyway. I've adapted it so that it is only one mission, instead of two like in the original PR.

~~The rest of the missions from #5112 may follow.~~

## Testing Done
I've tested this mission to ensure that the branch on the `to complete` node works. Although, if this could be a dialog box (something possible with #8504) it might be more in line with other missions.

### CODE REVIEW

File: data/human/human missions.txt
Comment (TheMarksman-ES) [inline data/human/human missions.txt:6883]:
```suggestion
			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, and every family member complained constantly about the lack of luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leaves without a word of thanks; they never even tell you their names. You wish you'd been paid more considering how terrible they were, although you're happy to be rid of them.`
```

**Review by TheMarksman-ES — CHANGES_REQUESTED** [review]

File: data/human/human missions.txt
Comment (lumbar527) [reply data/human/human missions.txt:6859]: Maybe the children share a bed?

File: data/human/human missions.txt
Comment (lumbar527) [inline data/human/human missions.txt:6883]: > **`You wish`**

I don't think you should tell the player what they think. Maybe this?
```suggestion
			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, and every member of the family complained constantly about the lack of luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even told you their names. You are finally rid of them, yet they only paid you a little compared to what you had to put up with.`
```

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:6859]: I'll change it to four. It was 3 in the original PR.
```suggestion
	passengers 4
```

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:6883]: It's not the first time that a like like "you're happy to be rid of them" appears... you also get "You are all too happy to return her to her owner" in the Pookie mission. I would think about changing it if it were a proper conversation, maybe? But, we might want to consider changing those instances later, too.

Comment (lumbar527) [reply data/human/human missions.txt]: OK.

Comment (bene-dictator) [reply data/human/human missions.txt]: I've changed it to try and remove that bit, how does it look now?

**Comment by Anarchist2** [comment]
We already have issues getting players to get to the Free Worlds in the first place; I'm not keen on adding a mission that takes them out of it at this point of time.

**Comment by bene-dictator** [comment]
Should I save this one for the future when we have campaigns for the Republic / Navy and Syndicate?

Or perhaps I could change it so that they get dropped off at the nearest Republic world after they complain about how long you're taking / bad accomodation?

**Comment by bene-dictator** [comment]
I'll wait for an answer, otherwise I'll close this PR in a few days if it's not likely to be merged soon. If it is going to wait until a Republic campaign is around, I'll close it until one is merged.

**Comment by Anarchist2** [comment]
I'll bring it up in the next dev chat.

File: data/human/human missions.txt
Comment (Amazinite) [inline data/human/human missions.txt:6863]: This condition doesn't exist. Checking for offered would be synonymous with checking if the mission were accepted (since you can't only either accept or defer the mission, and defer doesn't create any conditions).
```suggestion
		not "Rescue Katya 1: offered"
```

Although more generically, we could check if you've chosen sides.
```suggestion
		not "chosen sides"
```

**Review by Amazinite — COMMENTED** [review]
If we want to avoid leading the player away from the FW campaign with this mission, we could make it so that it only offers if you've started the Katya missions. So check if FW Katya 1 or FW Katya Alt 2 are done. (Or maybe more simply, we check the assisted FW condition.) That way you're at least already tied into the FW storyline by the time you're sent out, so you have a reason to return. The condition(s) that check if you've engaged with the FW prologue could then be removed when other campaigns are added.

**Comment by bene-dictator** [comment]
You have to deliberately accept the mission to get "chosen sides" while the "Rescue Katya 1: offered" condition isn't necessarily a deliberate choice by the player if they land on Bourne but don't want to accept the mission. So "chosen sides" seems more appropriate. Thanks!

**Comment by Amazinite** [comment]
You can't get the offered condition for that mission without also accepting the mission. That mission can only either `defer` or `accept`, and `defer` sets no conditions. The offered condition for a mission is only set if it was accepted or declined.

**Comment by bene-dictator** [comment]
Ah, whatever : )

File: data/human/human missions.txt
Comment (Amazinite) [inline data/human/human missions.txt:6894]:
```suggestion
```

File: data/human/human missions.txt
Comment (Amazinite) [inline data/human/human missions.txt:6887]: An endpoint at any point in an on complete conversation will end the conversation. No need for an empty label at the end of the conversation.
```suggestion
				decline
```

File: data/human/human missions.txt
Comment (Amazinite) [inline data/human/human missions.txt:6863]:
```suggestion
		not "chosen sides"
		has "assisted free worlds"
```

**Comment by TheGiraffe3** [comment]
@bene-dictator #8504 has been merged. Are you interested in doing, or having someone else (me), update the dialog to not be a conversation?

**Comment by bene-dictator** [comment]
I don’t mind, do whatever you like. I am still unable to review or create content for endless-sky right now.

**Pull request merged**
